Makefile

```
@@ -210,6 +210,28 @@ clients/cmd/promtail/promtail-debug:
 	CGO_ENABLED=$(PROMTAIL_CGO) go build $(PROMTAIL_DEBUG_GO_FLAGS) -o $@ ./$(@D)
 	$(NETGO_CHECK)
 
+#########
+# Mixin #
+#########
+
+MIXIN_PATH := production/loki-mixin
+MIXIN_OUT_PATH := production/loki-mixin-compiled
+
+loki-mixin:
+ifeq ($(BUILD_IN_CONTAINER),true)
+	$(SUDO) docker run $(RM) $(TTY) -i \
+		-v $(shell pwd):/src/loki$(MOUNT_FLAGS) \
+		$(IMAGE_PREFIX)/loki-build-image:$(BUILD_IMAGE_VERSION) $@;
+else
+	@rm -rf $(MIXIN_OUT_PATH) && mkdir $(MIXIN_OUT_PATH)
+	@cd $(MIXIN_PATH) && jb install
+	@mixtool generate all --output-alerts $(MIXIN_OUT_PATH)/alerts.yaml --output-rules $(MIXIN_OUT_PATH)/rules.yaml --directory $(MIXIN_OUT_PATH)/dashboards ${MIXIN_PATH}/mixin.libsonnet
+endif
+
+loki-mixin-check: loki-mixin
+	@echo "Checking diff:"
```

```suggestion
	@echo "Checking diff"
```
nit: We can remove `:` next git diff follows in the next line anyway.

It looks something like this now.
```
loki$ make loki-mixin-check 
docker run --rm --tty -i \
	-v /home/kavirajk/src/loki:/src/loki:delegated \
	grafana/loki-build-image:publish-compiled-mixin-a795b93-WIP loki-mixin;
make: Entering directory '/src/loki'
make: Leaving directory '/src/loki'
Checking diff:
diff --git a/production/loki-mixin-compiled/something.txt b/production/loki-mixin-compiled/something.txt
deleted file mode 100644
index e69de29bb..000000000
Please build mixin by running 'make loki-mixin'
make: *** [Makefile:233: loki-mixin-check] Error 1

```